lib/Cake/Utility/File.php

```diff
@@ -581,10 +581,11 @@ public function mime() {
  */
 	public function clearStatCache($all = false) {
 		if ($all === false && version_compare(PHP_VERSION, '5.3.0') >= 0) {
-			return clearstatcache(true, $this->path);
+			clearstatcache(true, $this->path);
+			return ;
```

Comment:
There is a space too much here before `;`.

Reply:
Sorry.
I mistook a rule with another project.